test/Microsoft.ML.FSharp.Tests/Microsoft.ML.FSharp.Tests.fsproj

```diff
@@ -5,6 +5,7 @@
     <PublicSign>false</PublicSign>
     <SourceLink></SourceLink>
     <PlatformTarget>$(TargetArchitecture)</PlatformTarget>
+    <LangVersion>4.7</LangVersion>
```

Comment:
Why not put this in Directory.Build.props and condition LangVersion based on the project extension? 4.7 for fsproj and 8.0 for csproj, that way if for some reason a new fsproj is added, it gets the right LangVersion and we have less duplication.

Reply:
Fixed with the latest update.